src/model/project.cpp

```diff
@@ -199,11 +199,7 @@ QString Project::detectEncoding() {
 	if (enc.empty()) {
```

Comment:
just an observation: looking at liblcf, the code above lacks converting from codepage:
https://github.com/EasyRPG/liblcf/blob/a55a9d4a45eeb36b73a37b6de736fb2f6adca718/src/reader_util.cpp#L218